src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -220,13 +220,15 @@ function configureTile(tile, layer, metadata, parent) {
     tile.updateMatrixWorld();
 }
 
-const textDecoder = new TextDecoder('utf-8');
 $3dTiles_Provider.prototype.executeCommand = function executeCommand(command) {
     const layer = command.layer;
     const metadata = command.metadata;
     const tile = new THREE.Object3D();
     configureTile(tile, layer, metadata, command.requester);
     const path = metadata.content ? metadata.content.url : undefined;
+    this._textDecoder = this._textDecoder || new TextDecoder('utf-8');
+    const textDecoder = this._textDecoder;
```

Comment:
`const textDecoder` is still useful?

Reply:
Yes it's used at lines 245 and 247

Reply:
you don't use `this._textDecoder` instead because of the context?

Reply:
yep